extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrPhotosImporter.java

```diff
@@ -74,6 +78,18 @@ public FlickrPhotosImporter(
     this.perUserRateLimiter = serviceConfig.getPerUserRateLimiter();
   }
 
+  public FlickrPhotosImporter(
+      AppCredentials appCredentials,
+      TemporaryPerJobDataStore jobStore,
+      Monitor monitor,
+      TransferServiceConfig serviceConfig,
+      IdempotentImportExecutor retryingIdempotentExecutor,
+      Boolean enableRetrying) {
```

Comment:
prefer primitive over boxed (also I think you're calling with the primitive anyway)

```suggestion
      boolean enableRetrying) {
```

Reply:
fixed, thanks